cnames_active.js

```diff
@@ -3121,6 +3121,7 @@ var cnames_active = {
   "uvcharts": "imaginea.github.io/uvCharts", // noCF? (don´t add this in a new PR)
   "v-charts": "elemefe.github.io/v-charts",
   "v-decorators": "v-decorators.netlify.com",
+  "v3.swup": "swup-v3-docs.netlify.app",
```

Comment:
```suggestion
  "v3.swup": "swup-v3-docs.netlify.app", // noCF
```
As this is a sub-subdomain, you'll need to have the Cloudflare proxy disabled, as Cloudflare's universal SSL only covers one level of subdomains.